index.html

```diff
@@ -2156,12 +2156,12 @@ <h2 id="presentations">Presentations</h2>
     },
     {
       "@context": "https://www.w3.org/ns/credentials/v2",
-      "id": "data:application/vc-ld+jwt;eyVjV...RMjU",
+      "id": "data:application/vc-ld+jwt;base64url,ebyVjV...RMjU",
```

Comment:
did a 'b' get added here accidentally or on purpose?

Reply:
nice spot, this was an accident

Reply:
fixed https://github.com/w3c/vc-jose-cose/pull/290/commits/4dbee0948a14a90e852cd7c7a47164eef095a7eb

Reply:
I don't think this is right, the `base64url` should not be present -- the media type here is already for a text-based format.

Reply:
@dlongley in what was merged I removed `base64url` and just kept `,`